### ISSUE TITLE
Minimun Transfer Duplicate Pokemon

### ISSUE THREAD

**Issue by ZaslavskiZ** [AnthonyOSX/GO-Bot#158]
Hi! I need an option that allow to set minium duplicate pokemon before transfer it. for example if i set 3 in minimum transfer duplicate pokemon, the bot only transfer a pokemon if has more than 3 duplicates.. thank you !!

**Comment by johnpyp** [comment]
Agreed!! Also, set a minimum for items before transfer, so list say: 20 pokeballs, 20 great balls, 20 ultra balls, 20 potions, 20 super potions, 20 hyper potions ect. Then the bot will only transfer down to the minimum.
